Legion.FrontEnd/src/events/EventsCalendar/Components/Calendar.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Header from './Header';
+import DaysOfWeek from './DaysOfWeek';
+import Month from './Month';
+import moment from 'moment';
```

Comment:
I won't make a huge deal about this because it is personal preference, but something I like to see is separation between libraries and internal project imports.  In my opinion, it makes it a little easier to find things.  So how I would make this is:
```js
import React from 'react';
import moment from 'moment';
                                // <-- space here
import Header from './Header';
import DaysOfWeek from './DaysOfWeek';
import Month from './Month';
```

Reply:
That should be fine.